docs/includes/usage-examples/FindOneTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
I get the following error when running this file:
`Could not open input file: vendor/bin/phpunit`

Reply:
Because there is no assertion.
You can add `$this->expectNotToPerformAssertions();` at the end of tests without assertion.

Reply:
I'm still getting the same message after adding `$this->expectNotToPerformAssertions()`.